### PR TITLE
Added !uploads/.uploads command

### PR DESCRIPTION
**Pull request by dongmaster**
It works similar to !phrases, except that it's for files that Parrot is storing.

### CODE REVIEW

File: volaparrot/commands/upload.py [hunk 1]
```diff
@@ -29,13 +29,15 @@
 from uuid import uuid4
 
 from path import path
+from io import BytesIO
```
Comment (RealDolos) [inline volaparrot/commands/upload.py]: put this to the std imports above

File: volaparrot/commands/upload.py [hunk 2]
```diff
@@ -29,13 +29,15 @@
...
+from time import time
```
Comment (RealDolos) [inline volaparrot/commands/upload.py]: same

File: volaparrot/commands/upload.py [hunk 3]
```diff
@@ -183,6 +188,29 @@ def cmd_unlockfile(self, remainder, msg):
         self.unlock_file(remainder)
         return True
 
+class FilesUploadCommand(UploadDownloadCommand, Command):
```
Comment (RealDolos) [inline volaparrot/commands/upload.py]: I know you just copied phrases stuff, but no need for a new command here, just add it in UploadDownloadCommand

**Comment by dongmaster** [comment]
Fixed, I hope.

File: volaparrot/commands/upload.py [hunk 4]
```diff
@@ -24,6 +24,8 @@
 # pylint: disable=bad-continuation,star-args,too-many-lines
 
 import logging
+from io import BytesIO
```
Comment (RealDolos) [inline volaparrot/commands/upload.py:27]: p-pls, seriously?

File: volaparrot/commands/upload.py [hunk 5]
```diff
@@ -24,6 +24,8 @@
...
+from time import time
 
 from collections import namedtuple
 from uuid import uuid4
```
Comment (RealDolos) [inline volaparrot/commands/upload.py:31]: belongs in this block ofc, nevermind, will fix when merging

Comment (dongmaster) [inline volaparrot/commands/upload.py:31]: oh woops

**Comment by RealDolos** [comment]
Closed by 94104aabc577266e8add73c137ab3354e6bc397b

**Pull request closed without merging**